src/main/java/io/littlehorse/quickstart/QuickstartWorkflow.java

```diff
@@ -1,30 +1,52 @@
 package io.littlehorse.quickstart;
 
+import io.littlehorse.sdk.common.proto.Comparator;
+import io.littlehorse.sdk.common.proto.VariableMutationType;
+import io.littlehorse.sdk.wfsdk.NodeOutput;
 import io.littlehorse.sdk.wfsdk.WfRunVariable;
 import io.littlehorse.sdk.wfsdk.Workflow;
 import io.littlehorse.sdk.wfsdk.WorkflowThread;
 
 public class QuickstartWorkflow {
 
-    public static final String WF_NAME = "quickstart";
-    public static final String GREET = "greet";
-
     /*
      * This method defines the logic of our workflow
      */
     public void quickstartWf(WorkflowThread wf) {
-        // Create an input variable, make it searchable
-        WfRunVariable name = wf.declareStr("input-name").searchable();
+        // Declare the input variables for the workflow
+        WfRunVariable firstName = wf.declareStr("firstName").searchable().required();
+        WfRunVariable lastName = wf.declareStr("lastName").searchable().required();
+        WfRunVariable ssn = wf.declareInt("ssn").masked().required();
+
+        // This variable will be used internally to track the identity verification
+        // status and will be able to search for any workflows that have the cusotmer's
+        // identity not verified
+        WfRunVariable identityVerified = wf.declareBool("identityVerified").searchable();
+
+        // Call the verify-identity task and retry it up to 3 times if it fails
+        wf.execute("verify-identity", firstName, lastName, ssn).withRetries(3);
+
+        // This is a blocking call, so it will wait until the event is posted or
+        // if the timeout is reached
+        NodeOutput identityPosted = wf.waitForEvent("identity-verified").timeout(60 * 60 * 24 * 3);
```

Comment:
```suggestion
        NodeOutput identityVerificationResult = wf.waitForEvent("identity-verified");
```

Reply:
We should wait for @mitchell-h 's input, but my vote: let's not put a timeout here unless we are going to put a failure handler on the timeout. Your `/learn` course could mention how to do that.

Either way, please rename to `identityVerificationResult`.